instrumentation/mongodb-reactive-streams-4.2/src/main/java/com/mongodb/reactivestreams/client/MongoCollection_Instrumentation.java

```diff
@@ -0,0 +1,464 @@
+            final BulkWriteOptions options) {
+
+        reportAsExternal(OP_BULK_WRITE);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<BulkWriteResult> bulkWrite(final ClientSession clientSession,
+            final List<? extends WriteModel<? extends TDocument>> requests,
+            final BulkWriteOptions options) {
+        reportAsExternal(OP_BULK_WRITE);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<InsertOneResult> insertOne(final TDocument document, final InsertOneOptions options) {
+        reportAsExternal(OP_INSERT_ONE);
+        return Weaver.callOriginal();
+    }
+
+    @Trace(leaf = true)
+    public Publisher<InsertOneResult> insertOne(final ClientSession clientSession, final TDocument document,
+            final InsertOneOptions options) {
+        reportAsExternal(OP_INSERT_ONE);
+        return Weaver.callOriginal();
+    }
+
+    @Trace(leaf = true)
+    public Publisher<InsertManyResult> insertMany(final List<? extends TDocument> documents, final InsertManyOptions options) {
+        reportAsExternal(OP_INSERT_MANY);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<InsertManyResult> insertMany(final ClientSession clientSession, final List<? extends TDocument> documents,
+            final InsertManyOptions options) {
+        reportAsExternal(OP_INSERT_MANY);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<DeleteResult> deleteOne(final Bson filter, final DeleteOptions options) {
+        reportAsExternal(OP_DELETE_ONE);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<DeleteResult> deleteOne(final ClientSession clientSession, final Bson filter,
+            final DeleteOptions options) {
+        reportAsExternal(OP_DELETE_ONE);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<DeleteResult> deleteMany(final Bson filter, final DeleteOptions options) {
+
+        reportAsExternal(OP_DELETE_MANY);
+        return Weaver.callOriginal();
+    }
+
+    @Trace(leaf = true)
+    public Publisher<DeleteResult> deleteMany(final ClientSession clientSession, final Bson filter,
+            final DeleteOptions options) {
+
+        reportAsExternal(OP_DELETE_MANY);
+        return Weaver.callOriginal();
+    }
+
+    @Trace(leaf = true)
+    public Publisher<UpdateResult> replaceOne(final Bson filter, final TDocument replacement,
+            final ReplaceOptions options) {
+        reportAsExternal(OP_REPLACE_ONE);
+        return Weaver.callOriginal();
+
+    }
+
+    @Trace(leaf = true)
+    public Publisher<UpdateResult> replaceOne(final ClientSession clientSession, final Bson filter,
+            final TDocument replacement,
+            final ReplaceOptions options) {
+
```

Comment:
```suggestion
```
I suspect it is a formatting rule for method signatures that are broken up onto multiple lines, as this one seems consistent. For the single line method signatures it does not.